celery/backends/database/__init__.py

```diff
@@ -68,18 +68,17 @@ class DatabaseBackend(BaseBackend):
     # to not bombard the database with queries.
     subpolling_interval = 0.5
 
-    task_cls = Task
```

Comment:
what this `cls` means? is reserved name?

Reply:
It's just a class variable used to manufacture tasks.